### PR TITLE
deploy to single existing device

### PR DESCRIPTION
**Pull request by alanrenmsft**
image

image

### CODE REVIEW

File: extensions/asde-deployment/notebooks/edge/deploy-sql-edge-single-device.ipynb [hunk 1]
```diff
@@ -0,0 +1,365 @@
...
+    },
+    "nbformat_minor": 2,
+    "nbformat": 4,
+    "cells": [
+        {
+            "cell_type": "markdown",
+            "source": [
+                "![Microsoft](https://raw.githubusercontent.com/microsoft/azuredatastudio/main/extensions/resource-deployment/images/microsoft-small-logo.png)\n",
+                "## Deploy Azure SQL Edge to an existing device via IoT hub\n",
+                "This notebook will walk you through process of deploying Azure SQL Edge to an existing device of an IoT hub\n",
```
Comment (Benjin) [inline extensions/asde-deployment/notebooks/edge/deploy-sql-edge-single-device.ipynb:28]: probably want a colon at the end

`IoT hub:\n",`

File: extensions/asde-deployment/notebooks/edge/deploy-sql-edge-single-device.ipynb [hunk 2]
```diff
@@ -0,0 +1,365 @@
...
+                "1. Deploy SQL Edge module to the device with optional dacpac\n",
+                "1. If the dacpac is selected, an storage account will be created to host the dacpac file\n",
```
Comment (Benjin) [inline extensions/asde-deployment/notebooks/edge/deploy-sql-edge-single-device.ipynb:30]: typo: **a** storage account

File: extensions/asde-deployment/package.nls.json [hunk 3]
```diff
@@ -35,6 +35,12 @@
...
 	"dacpac_path": "Dacpac zip file",
 	"azure-info-section-title": "Azure information",
-	"sqlserver-info-section-title": "SQL Server information",
-	"dacpac-zip-files": "Dacpac zip files"
+	"sqlserver-info-section-title": "Azure SQL Edge information",
+	"dacpac-zip-files": "Dacpac zip files",
+	"sql-edge-azure-single-device-display-name": "Existing device of an Azure IoT Hub",
+	"sql-edge-azure-single-device-title": "Deploy Azure SQL Edge to an existing device",
+	"iot-hub-name": "IoT Hub name",
+	"device-id": "Device Id",
```
Comment (Benjin) [inline extensions/asde-deployment/package.nls.json:43]: d in "ID" should be capitalized when it's getting displayed in UI/string

Comment (alanrenmsft) [reply extensions/asde-deployment/package.nls.json:43]: @Benjin Thanks for the careful review.

**Pull request merged**